gradle/versions.gradle

```diff
@@ -1,44 +1,42 @@
 ext {
     javaVersion = JavaVersion.VERSION_1_8
-    gradleVersion = '7.5.1'
+    gradleVersion = '8.3'
     squitVersion = '5.1.0'
 
-    gradlePublishVersion = '1.1.0'
+    gradlePublishVersion = '1.2.1'
 
-    kotlinVersion = '1.7.21'
-    dokkaVersion = '1.7.20'
+    kotlinVersion = '1.9.10'
+    dokkaVersion = '1.8.20'
 
-    jacocoVersion = '0.8.8'
-    gradleVersionsVersion = '0.44.0'
-    gradleTestIdeaFixVersion = '0.1.0'
+    jacocoVersion = '0.8.10'
+    gradleVersionsVersion = '0.47.0'
 
-    jaxenVersion = '1.2.0'
-    dom4jVersion = '2.1.3'
+    jaxenVersion = '2.0.0'
+    dom4jVersion = '2.1.4'
     pullParserVersion = '2.1.10'
-    gsonVersion = '2.10'
-    xmlSecVersion = '3.0.1'
-    xmlUnitVersion = '2.9.0'
-    jsonUnitVersion = '2.36.0'
+    gsonVersion = '2.10.1'
+    xmlSecVersion = '3.0.2'
+    xmlUnitVersion = '2.9.1'
+    jsonUnitVersion = '2.38.0'
     typesafeConfigVersion = '1.4.2'
-    okhttpVersion = '4.10.0'
-    kotlinHtmlVersion = '0.8.0'
+    okhttpVersion = '4.11.0'
+    kotlinHtmlVersion = '0.9.1'
     diffUtilsVersion = '3.0'
     alphanumericComparatorVersion = "1.4.1"
 
-    bootstrapVersion = '4.6.0'
-    jqueryVersion = '3.6.0'
+    bootstrapVersion = '5.3.1'
```

Comment:
If we want to upgrade this we also need to update the frontend since this version is not backwards compatible.